lib/jobs/install-os.js

```diff
@@ -236,6 +237,18 @@ function installOsJobFactory(
                     }
                 }
             });
+
+            self._subscribeTaskNotification(function(data) {
+                assert.object(data);
+                if(
+                    data.nodeId === self.nodeId
+                    && data.taskId === self.taskId
+                    && data.type === 'osInstall'
```

Comment:
I have some concern to judge the `type` here.
- the taskId is enough to tell this is install os job;
- this makes  the code cannot be shared by other job that also wants notification, for example, for wait-completion-uri job, to define another type "waitCompletionUri"? I think it's not worth.